services/app-api/src/handlers/third_party_API_authorizer.ts

```diff
@@ -0,0 +1,59 @@
+import type {
+    APIGatewayAuthorizerResult,
+    PolicyDocument,
+    Handler,
+} from 'aws-lambda'
+import { newJWTLib } from '../jwt'
+import { AuthenticationError } from 'apollo-server-lambda'
+
+// Hard coding this for now, next job is to run this config to this app.
```

Comment:
This is a duplication from what was temporarily added in `apollo_gql.ts`. But necessary here to call its `userIDFromToken` method